### ISSUE TITLE
“Old” unread notifications do not show up

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#398]
Is this a limitation of the API or a bug?

I have one issue not marked as read because I wanted to get back to it, but it seems to not be loaded in the unread section anymore. I can find it (in its unread state) when loading a few pages in All, but not in Unread (not even when loading the (phantom?) next page(s).
![5831a4f0-9bb2-4e18-9c7a-c2e604cf5d8a](https://user-images.githubusercontent.com/4190298/30776286-332a1d5c-a0a4-11e7-8c51-11649a0d2e17.png)

**Comment by rnystrom** [comment]
Yup, limitation of the V3 notifications endpoint unfortunately. Same reason the "view next page" stuff is so flaky.

**Comment by BasThomas** [comment]
Lol, seems to be broken on GitHub.com also 😬
